parsons/zoom/zoom.py

```diff
@@ -225,3 +272,103 @@ def get_webinar_registrants(self, webinar_id):
         tbl = self._get_request(f"webinars/{webinar_id}/registrants", "registrants")
         logger.info(f"Retrieved {tbl.num_rows} webinar registrants.")
         return tbl
+
+    def get_meeting_poll(self, meeting_id: str, poll_id: str) -> Table:
+        """
+        Get a specific poll for a given meeting ID
+
+        `Args`:
+            meeting_id: str
+                Unique identifier for Zoom meeting
+            poll_id: str
+                Unique identifier for poll
+
+        `Returns`:
+            Parsons Table of all polling responses
+        """
+
+        endpoint = f"meetings/{meeting_id}/polls/{poll_id}"
+        tbl = self._get_request(endpoint=endpoint, data_key="polls")
+
+        if type(tbl) == dict:
```

Comment:
The return type is a parsons.Table, but here we're returning a dict. Should this just return `Table()` here?

Same comment applies for the other poll functions.